app/views/email_notifications/welcome_attendance.en.html.haml

```diff
@@ -7,11 +7,11 @@ Hi #{h(@attendance.full_name)},
 %ul
   %li Bring a valid document
   %li If you are registered as student, you’ll be invited to comprove it by show your student id. You'll be asked to pay the full price if you don't have it.
-  %li The welcome coffee and the registration will be in place at 9:00
+  %li The welcome coffee and the registration will be in place at #{@attendance.event.start_date.to_date.strftime('%d/%m/%Y')}
```

Comment:
Should this have been like for the PT one with HH:mm instead of %d/%M/%Y?
Also... US English would do %M/%d/%Y